pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -107,6 +105,7 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = with qt6Packages; [
     curl
     ffmpeg
+    fontconfig
```

Comment:
Will this make the `programs.ladybird.enable` option obsolete?

Reply:
I don't know enough about how NixOS, fontconfig, and fonts.fontDir.enable option interact to be able to say for sure, I just know that fontconfig is definitely a dependency for ladybird on macos as of https://github.com/LadybirdBrowser/ladybird/commit/dd1b8de671747f0a5efc84b77933e13208136507 and seems to have always been a dependency on linux.